events-manager.js

```diff
@@ -201,7 +199,7 @@
 
       _fireCallback(dones, callback, results);
 
-      return values;
+      return results;
```

Comment:
It will be populated by all the `.onAsync()` functions, but may return an empty `Array` for sure if none were bound to this event